### PR TITLE
add update option for consultation 

### PR DESCRIPTION
**Pull request by bodhisha**

### CODE REVIEW

File: src/Components/Facility/ConsultationDetails.tsx [hunk 1]
```diff
@@ -14,147 +14,185 @@ import { DailyRoundsModel } from "../Patient/models";
...
-    );
+  const fetchData = useCallback(
+    async (status: statusType) => {
+      setIsLoading(true);
+      const res = await dispatch(getConsultation(consultationId));
+      if (!status.aborted) {
+        if (res && res.data) {
+          const data: ConsultationModel = {
+            ...res.data,
+            symptoms_text: "",
```
Comment (tomahawk-pilot) [inline src/Components/Facility/ConsultationDetails.tsx:43]: Codacy found an issue: [Identifier 'symptoms_text' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)

**Comment by tomahawk-pilot** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160&bid=19014370)

File: src/Components/Facility/ConsultationDetails.tsx [hunk 2]
```diff
@@ -14,147 +14,185 @@ import { DailyRoundsModel } from "../Patient/models";
...
+                ?.text || res.data.category,
+          };
+          if (res.data.symptoms && res.data.symptoms.length) {
+            const symptoms = res.data.symptoms
+              .filter((symptom: number) => symptom !== 9)
+              .map((symptom: number) => {
+                const option = symptomChoices.find((i) => i.id === symptom);
+                return option ? option.text.toLowerCase() : symptom;
+              });
+            data.symptoms_text = symptoms.join(", ");
```
Comment (tomahawk-pilot) [inline src/Components/Facility/ConsultationDetails.tsx:55]: Codacy found an issue: [Identifier 'symptoms_text' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)

**Comment by dauntlessnomad** [comment]
@bodhisha fix the conflicts. and reference the issue.

**Comment by dauntlessnomad** [comment]
#384

**Comment by dauntlessnomad** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160&bid=19014370)

File: src/Components/Facility/ConsultationDetails.tsx [hunk 3]
```diff
@@ -14,228 +14,348 @@ import { DailyRoundsModel } from "../Patient/models";
...
-    );
+  const fetchData = useCallback(
+    async (status: statusType) => {
+      setIsLoading(true);
+      const res = await dispatch(getConsultation(consultationId));
+      if (!status.aborted) {
+        if (res && res.data) {
+          const data: ConsultationModel = {
+            ...res.data,
+            symptoms_text: "",
```
Comment (dauntlessnomad) [inline src/Components/Facility/ConsultationDetails.tsx:43]: Codacy found an issue: [Identifier 'symptoms_text' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)

File: src/Components/Facility/ConsultationDetails.tsx [hunk 4]
```diff
@@ -14,228 +14,348 @@ import { DailyRoundsModel } from "../Patient/models";
...
+                ?.text || res.data.category,
+          };
+          if (res.data.symptoms && res.data.symptoms.length) {
+            const symptoms = res.data.symptoms
+              .filter((symptom: number) => symptom !== 9)
+              .map((symptom: number) => {
+                const option = symptomChoices.find((i) => i.id === symptom);
+                return option ? option.text.toLowerCase() : symptom;
+              });
+            data.symptoms_text = symptoms.join(", ");
```
Comment (dauntlessnomad) [inline src/Components/Facility/ConsultationDetails.tsx:55]: Codacy found an issue: [Identifier 'symptoms_text' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)

File: src/Components/Facility/ConsultationDetails.tsx [hunk 5]
```diff
@@ -14,228 +14,348 @@ import { DailyRoundsModel } from "../Patient/models";
...
-                                    onClick={(e) => navigate(`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/daily-rounds/${itemData.id}`)}
-                                >View Daily Rounds Details</Button>
-                            </div>
-                        </div>
-                    </div>
-                </div >
-            );
-        });
-    }
+  let roundsList: any;
```
Comment (dauntlessnomad) [inline src/Components/Facility/ConsultationDetails.tsx:161]: Codacy found an issue: ['roundsList' is already defined.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)

**Comment by bodhisha** [comment]
Fixed the conflicts

**Pull request merged**